Documentation/supermuc.rst

```diff
@@ -177,3 +181,267 @@ set compiler options, run cmake, and compile with:
   echo $SLURM_NTASKS
   ulimit -Ss 2097152
   mpiexec -n $SLURM_NTASKS SeisSol_Release_sskx_4_elastic parameters.par
+
+
+Marconi 100
+-----------
+
+Marconi 100 is a distributed multi-GPU HPC system equipped with 4 Nvidia V100 GPUs
+and 2 IBM Power9 CPUs per node. This architecture usually comes with a pre-installed
+CUDA-Aware Spectrum-MPI. However, SeisSol cannot operate with Spectrum-MPI because 
+of GPU memory leaks caused by this MPI implementation. This part of documentation
+describes how to setup and configure OpenMPI 4.1.x together with UCX 1.10.x to operate
+on IBM Power and similar HPC systems. 
+
+Installing Main Libraries and Packages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+1. Create the following launch file:
+
+::
+
+  $ touch $HOME/launch.sh
+  $ cat $HOME/launch.sh
+  #!/bin/bash
+
+  VERSION="openmpi4-1-x_ucx1-10-x"
+  export SEISSOL_INSTALL=$HOME/usr/local/${VERSION}
+
+  export PATH=$SEISSOL_INSTALL/bin:$PATH
+  export LIBRARY_PATH=$SEISSOL_INSTALL/lib:$LIBRARY_PATH
+  export LD_LIBRARY_PATH=$SEISSOL_INSTALL/lib:$LD_LIBRARY_PATH
+  export PKG_CONFIG_PATH=$SEISSOL_INSTALL/lib/pkgconfig:$PKG_CONFIG_PATH
+  export CMAKE_PREFIX_PATH=$SEISSOL_INSTALL
+  export CPATH=$SEISSOL_INSTALL/include:$CPATH
+  export CPLUS_INCLUDE_PATH=$SEISSOL_INSTALL/include:$CPLUS_INCLUDE_PATH
+  export C_INCLUDE_PATH=$SEISSOL_INSTALL/include:$C_INCLUDE_PATH
+
+2. Load basic modules, source launch file and create an install directory:
+
+::
+
+  $ source ./launch.sh
+  $ mkdir -p $SEISSOL_INSTALL
+
+  $ module load gnu/8.4.0
+  $ module load cmake/3.20.0
+  $ module load python/3.8.2
+
+3. Create any directory where you are going to configure and build libraries and packages. For example, 
+
+::
+
+  mkdir -p $HOME/Download
+  cd $HOME/Download
+
+4. Install UCX:
+
+:: 
```

Comment:
it seems this block is not recognized as a code block. (but maybe its a problem with GitHub code highlighting)

Reply:
Fixed.